lib/cli-create-embed-token.ts

```diff
@@ -51,7 +52,8 @@ export default function CliCreateEmbedToken() {
             if (!(settings.report || settings.dataset)) {
                 return cli.error('Either reportId or datasetId must be set');
             }
-            token = powerbi.PowerBIToken.createReportEmbedToken(settings.collection, settings.workspace, settings.report, settings.dataset, settings.scopes, settings.username, settings.roles);
+            let expiration = settings.expiration ? new Date(settings.expiration) : void 0;
+            token = powerbi.PowerBIToken.createReportEmbedToken(settings.collection, settings.workspace, settings.report, settings.dataset, settings.scopes, settings.username, settings.roles, expiration);
```

Comment:
`expiration` parameter of `createReportEmbedToken` should be Date object or no value (undefined, void 0). In fact, `expiration` is initialized to null by 'createReportEmbedToken' if it's undefined, but it's better to observe function parameter type, IMHO.